fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -299,6 +304,42 @@ public void alterMaterializedViewStatus(MaterializedView materializedView, Strin
         }
     }
 
+    /**
+     * Try to activate the new mv. Throw exception if fail
+     */
+    private static QueryStatement tryActiveMV(MaterializedView materializedView, ConnectContext context) {
```

Comment:
it seems that there is no logic in this method that try to active mv, so use another name?

Reply:
+1

Reply:
done